### PR TITLE
Passwordless Login For Your PHP Project

### PR DESCRIPTION
**Pull request by oyedeletemitope**
#### Please note that in case of below checklist not updated accordingly, the maintainers have the right to immediately close your pull request.

- [ ] You created a branch from `dev` branch.
- [ ] You PR is raised on `dev` branch and not on`master`.
- [ ] Blog follows [Blog Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/GUIDELINES.md).
- [ ] You have read the [Contribution Guidelines](https://github.com/LoginRadius/engineering-portal/blob/master/CONTRIBUTING.md) before creating this PR.
- [ ] Checked for any duplicate blog entry.
- [ ] Checked for all grammatical errors in the blog.
- [ ] You have not published the blog content anywhere else on the internet on platforms like medium, dev.to etc.
- [ ] The content is not copy-pasted from anywhere else on the Internet.

### CODE REVIEW

File: content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+                  </button>  
+               </div>
+            </form>
+         </div>
+      </div>
+   </body>
+</html>
+```
+
+For `verify.php`, add the code as follows:
```
Comment (raghunath-r-a) [inline content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md:206]: @oyedeletemitope

Please add some description as to what the code programmatically does. And please feel free to add inline comments in the code. Please do the same for the remaining .PHP files wherever you have mentioned and added the code.

File: content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>welcome back user</h1>
+</body>
+</html>
+```
+
+You can now run application and test it. That's how you can implement passwordless authentication using LoginRadius for your PHP project.
```
Comment (raghunath-r-a) [inline content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md:423]: @oyedeletemitope Please add a screenshot showing that the authentication is successful

File: content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md [hunk 3]
```diff
@@ -0,0 +1,429 @@
...
+
+### Conclusion
```
Comment (raghunath-r-a) [inline content/engineering/guest-post/passwordless-authentication-for-your-php-project/index.md:425]: @oyedeletemitope For easier reference, please add all the sample code used in this tutorial by raising a separate PR here: https://github.com/LoginRadius/engineering-blog-samples

**Review by raghunath-r-a — COMMENTED** [review]
@oyedeletemitope I have done the PHASE-I review and added some suggestions. Please review them and add the changes so that we can take it forward for the next phase of review. 

Thank you, 
@raghunath-r-a

**Comment by oyedeletemitope** [comment]
Hello @raghunath-r-a , I have made changes as requested and also uploaded my codes

**Comment by raghunath-r-a** [comment]
Hi @oyedeletemitope I have previously foce-pushed the changes so that it is in line with our new blog structure. It seems like your new push has disrupted everything. 

Please get a fresh copy of the `develop` branch from the original repo here: https://github.com/LoginRadius/engineering-portal/tree/develop

And then please apply the commits I have created: 

- [rebase with new blog structure](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/eec8ada5c7a808ee772ca2d01e4953fa649bc396)

- [initial checks](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/3e976f2919f65c95f8e695aeb0da45678b503dda)

- [initial checks - 2](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/3737b5c31ccfeaeef482d86f5d2fbc04c7fcd32f)

And then add your new changes on top of them. This will help to ensure your PR and changes are consistent. And helps us take this to the next phase. 

Please share with us any questions you have.

Best regards,
@raghunath-r-a

**Comment by oyedeletemitope** [comment]
Hello @raghunath-r-a , do you mean I should create a new or from that repo after I effect those changes?

**Comment by raghunath-r-a** [comment]
@oyedeletemitope 

You can close this PR and raise a new one. 

Else, you can rebase/reinitiate your `master` branch with `develop`. This is necessary as we have updated our entire publication with improvements.

As long as the following 3 commits are intact and then your new changes are added, it will be good to go:

[rebase with new blog structure](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/eec8ada5c7a808ee772ca2d01e4953fa649bc396)

[initial checks](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/3e976f2919f65c95f8e695aeb0da45678b503dda)

[initial checks - 2](https://github.com/LoginRadius/engineering-portal/pull/1010/commits/3737b5c31ccfeaeef482d86f5d2fbc04c7fcd32f)

**Comment by oyedeletemitope** [comment]
Hello @raghunath-r-a , I have made the changes and added yours too. I'll close this pr and push them into a new pr

**Pull request closed without merging**